src/components/item-card/ItemCard.tsx

```diff
@@ -13,12 +13,21 @@ type Props = {
   isFavorite: boolean
 }
 
+const MAX_DESCRIPTION_LENGTH = 69
+
 const ItemCard: FC<Props> = ({ item, isFavorite }) => {
   const [selectedWeight, setSelectedWeight] = useState(item.prices[0].weight)
   const [selectedPrice, setSelectedPrice] = useState(item.prices[0].price)
 
   const dispatch = useDispatch()
 
+  const truncatedDescription =
+    item.description.length > MAX_DESCRIPTION_LENGTH
+      ? `${item.description.slice(0, MAX_DESCRIPTION_LENGTH)}...`
+      : item.description
```

Comment:
you can do this without state, just using css
ref https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/